### PR TITLE
Indicate that Cliquet is now deprecated

### PR DESCRIPTION
**Pull request by glasserc**
Following what I saw at https://github.com/citusdata/pg_shard, I added a deprecation notice to the README. Also, following http://stackoverflow.com/questions/24085400/how-to-formally-deprecate-a-pip-package, add a warning when the package is imported.

We can merge this once the tests pass on my fork of Kinto and I merge that into master.

### CODE REVIEW

File: cliquet/__init__.py [hunk 1]
```diff
@@ -16,6 +16,13 @@
 from cliquet.logs import logger
 
 
+import warnings
+warnings.warn("cliquet is now deprecated. Please update to kinto.core at "
```
Comment (Natim) [inline cliquet/__init__.py]: nit: Cliquet with a capital C.

**Comment by Natim** [comment]
r+ (I won't ask for a test to make sure the warning is displayed.)

File: cliquet/__init__.py (hunk 1 above)
Comment (glasserc) [inline cliquet/__init__.py]: I wasn't sure because the package name is `cliquet`, but the library's name is `Cliquet`.

**Comment by Natim** [comment]
Actually while running cliquet with this code, I do not see the warning.

File: cliquet/__init__.py [hunk 2]
```diff
@@ -16,6 +16,13 @@
 from cliquet.logs import logger
 
 
+import warnings
+warnings.warn("Cliquet is now deprecated. Please update to kinto.core at "
```
Comment (Natim) [inline cliquet/__init__.py]: It seems that ``DeprecationWarning`` are ignored by default so we need to run ``python -Walways`` to see them.
Maybe we could use ``logger.warn`` instead or both?

**Comment by glasserc** [comment]
Deprecation warnings are apparently silenced by default :(. Added a call to logger.warn in 8d20e3ba62de7eaa3e5787b1902ff0090325bf66.

**Pull request merged**